lib/test/cmaf/ham/init.ts

```diff
@@ -0,0 +1,34 @@
+import { setDashParser, setDashSerializer, setHlsParser, type DashManifest, type HlsManifest } from '@svta/common-media-library';
+import { Parser } from 'm3u8-parser';
+import { Builder, parseString } from 'xml2js';
+
+setDashParser((raw: string) => {
+	let parsed: DashManifest;
+	parseString(raw, (err: Error | null, result: DashManifest) => {
+		if (err) {
+			throw new Error(err.message);
+		}
+		parsed = result as DashManifest;
+	});
+	// @ts-ignore
```

Comment:
Why do you need ts-ignore? If it's still useful (unmatched types), what about replacing it with ts-expect-error?

```suggestion
	// @ts-expect-error
```

I also encourage adding the reason after the comment, with the why

Reply:
The error it's ignoring is: `Variable 'parsed' is used before being assigned.ts(2454)`. `parsed` is populated by the synchronous `parseString` callback.